mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -63,7 +63,6 @@ HVR:
 	Mobile:
 		Speed: 99
 		Locomotor: hover
-		RequiresCondition: !empdisable
```

Comment:
Why are you removing this? The hover tank should be just as emp-able and capturable as any other vehicle.

Reply:
I suspect this was already inherited from the default template?